go.mod

```diff
@@ -10,6 +10,7 @@ require (
 	github.com/banzaicloud/istio-operator v0.0.0-20200407070503-3f7dc6953a7b
 	github.com/banzaicloud/k8s-objectmatcher v1.4.1
 	github.com/banzaicloud/kafka-operator/api v0.0.0
+	github.com/banzaicloud/kafka-operator/pkg/sdk v0.4.1
```

Comment:
I think this should not be here. You should use the `github.com/banzaicloud/kafka-operator/api` package instead of any occurrence of the `github.com/banzaicloud/kafka-operator/pkg/sdk` package.